tests/test_cli.py

```diff
@@ -21,9 +20,14 @@ def __init__(self, returncode, cmd, output):
 
 def run(*args, **kwargs):
     command = kwargs.pop('command', 'grip')
+    which_grip = Popen(["which", command], stdout=PIPE)
+    which_grip.communicate()
+    if which_grip.returncode:
+        cmd = ['python', 'grip/__main__.py'] + list(args)
+    else:
+        cmd = [command] + list(args)
```

Comment:
`test_cli` is intended to be an "end-to-end" test. So installing with `pip install -e .[tests]` is a prerequisite for running the tests. No need to add a conditional here.